tests/dashboards/commands_tests.py

```diff
@@ -78,79 +82,148 @@ def test_export_dashboard_command(self, mock_g1, mock_g2):
+                    "meta": {"height": 50, "sliceName": "Treemap", "width": 8},
                     "type": "CHART",
                 },
+                "COLUMN-071bbbad": {
+                    "children": ["ROW-1e064e3c", "ROW-afdefba9"],
+                    "id": "COLUMN-071bbbad",
+                    "meta": {"background": "BACKGROUND_TRANSPARENT", "width": 9},
+                    "type": "COLUMN",
+                },
+                "COLUMN-fe3914b8": {
+                    "children": ["CHART-36bfc934", "CHART-37982887"],
+                    "id": "COLUMN-fe3914b8",
+                    "meta": {"background": "BACKGROUND_TRANSPARENT", "width": 2},
+                    "type": "COLUMN",
+                },
+                "GRID_ID": {
+                    "children": ["ROW-46632bc2", "ROW-3fa26c5d", "ROW-812b3f13"],
+                    "id": "GRID_ID",
+                    "type": "GRID",
+                },
+                "HEADER_ID": {
+                    "id": "HEADER_ID",
+                    "meta": {"text": "World's Bank Data"},
+                    "type": "HEADER",
+                },
+                "ROOT_ID": {"children": ["GRID_ID"], "id": "ROOT_ID", "type": "ROOT"},
+                "ROW-1e064e3c": {
+                    "children": ["COLUMN-fe3914b8", "CHART-2d5b6871"],
+                    "id": "ROW-1e064e3c",
+                    "meta": {"background": "BACKGROUND_TRANSPARENT"},
+                    "type": "ROW",
+                },
+                "ROW-3fa26c5d": {
+                    "children": ["CHART-b5e05d6f", "CHART-0fd0d252"],
+                    "id": "ROW-3fa26c5d",
+                    "meta": {"background": "BACKGROUND_TRANSPARENT"},
+                    "type": "ROW",
+                },
+                "ROW-46632bc2": {
+                    "children": ["COLUMN-071bbbad", "CHART-17e0f8d8"],
+                    "id": "ROW-46632bc2",
+                    "meta": {"background": "BACKGROUND_TRANSPARENT"},
+                    "type": "ROW",
+                },
+                "ROW-812b3f13": {
+                    "children": ["CHART-a4808bba", "CHART-e76e9f5f"],
+                    "id": "ROW-812b3f13",
+                    "meta": {"background": "BACKGROUND_TRANSPARENT"},
+                    "type": "ROW",
+                },
+                "ROW-afdefba9": {
+                    "children": ["CHART-2ee52f30", "CHART-97f4cb48"],
+                    "id": "ROW-afdefba9",
+                    "meta": {"background": "BACKGROUND_TRANSPARENT"},
+                    "type": "ROW",
+                },
                 "DASHBOARD_VERSION_KEY": "v2",
             },
-            "metadata": {
-                "timed_refresh_immune_slices": [],
-                "expanded_slices": {},
-                "refresh_frequency": 0,
-                "default_filters": "{}",
-                "color_scheme": None,
-            },
+            "metadata": {"mock_key": "mock_value"},
```

Comment:
Why the change?

Reply:
I think the old value was the side effect of some other tests. Now I set `{"mock_key": "mock_value"}` explicitly in the fixture.